README.md

```diff
@@ -0,0 +1,51 @@
+# To-Do App
+
+## Description
+
+In this project I built a to do app using plain JavaScript.You can make a new project and inside each project make a task, you can set the task priority and due dates.
+You can also delete each task or project.I used LocalStorage for storing the data and uuid for generating random id's.
+
+## Built With
+
+- JavaScript
+- Webpack
+- Bootstrap
+
+## Set up this project locally
+
+- Open the terminal in the directory where you want to install the app;
+- Run command `https://github.com/MarkoNS1990/to_do_app` which will clone the project;
+- Switch to the main directory of project using cd;
+- Run `npm install webpack webpack-cli --save-dev` in your computer terminal to install Webpack (you will need Node.js installed on your machine);
+- Open index.html file in your browser;
+- Use it.
```

Comment:
- [ ] Kindly also instruct the user to install the project packages with `npm/yarn install`.